frontend/src/components/regions/UsersAndGroups/utils.js

```diff
@@ -0,0 +1,27 @@
+const timeNow = new Date()
+const milliseconds = 1000 * 60 * 60 * 24
+const timeDifferenceInDays = (time) => {return Math.ceil(Math.abs(timeNow - new Date(time)) / milliseconds)}
+let prColor
+
+export const getPrColor = (oldestOpenPrTime) => {
+    if (timeDifferenceInDays(oldestOpenPrTime) === 1){
+        prColor = "black"
+    }else if(timeDifferenceInDays(oldestOpenPrTime) <= 2) {
+        prColor = "orange"
+    }else {
+        prColor = "red"
+    }
```

Comment:
On PR #272 I created a `color.js` file , working with it here would make things DRYer, but that PR has not been merged, I guess this is good for now

Reply:
Alright